File: php/Mahasiswa.php
<!-- Saya Amida Zulfa Laila NIM 2101147 mengerjakan Latihan Praktikum 1
dalam mata kuliah Desain dan Pemrograman Berorientasi Objek untuk keberkahanNya maka
saya tidak melakukan kecurangan seperti yang telah dispesifikasikan.
Aamiin. -->
<?php
// Class declaration 
class Mahasiswa {
    // atribut private
    private $nama = '';
    private $nim = '';
    private $prodi = '';
    private $fakultas = '';
    private $foto = '';

    //constructor
    public function __construct( $nama = '', $nim = '', $prodi='', $fakultas='', $foto='' ) {
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
        $this->foto = $foto;
    }

    // Getter dan Setter
    //set nama
    public function setNama($nama){
        $this->nama = $nama;
    }
    //get nama
    public function getNama(){
        return $this->nama;
    }
    //set nim
 
    public function setNim($nim){
        $this->nim = $nim;
    }
    //get nim

    public function getNim(){
        return $this->nim;
    }   
    //set prodi
    public function setProdi($prodi){
        $this->prodi = $prodi;
    }
    //get prodi
    public function getProdi(){
        return $this->prodi;
    } 
    //set fakultas
    public function getfakultas(){
        return $this->fakultas;
    }  
    //get fakultas
    public function setFakultas($fakultas){
        $this->fakultas = $fakultas;
    }
    //set foto
    public function getFoto(){
        return $this->foto;
    }
    //get foto
    public function setFoto($foto){
        $this->foto = $foto;
    }
}

?>

File: java/Mahasiswa.java
/* Saya Amida Zulfa Laila NIM 2101147 mengerjakan Latihan Praktikum 1
dalam mata kuliah Desain dan Pemrograman Berorientasi Objek untuk keberkahanNya maka
saya tidak melakukan kecurangan seperti yang telah dispesifikasikan.
Aamiin. */

/* Deklarasi Class Mahasiswa */
public class Mahasiswa
{
    // Private attributes
    private String nama;
    private int nim;
    private String prodi;
    private String fakultas;

     /* contructor */
    public Mahasiswa()
    {
    }

    // Constructor dengan parameter
    public Mahasiswa(String nama, int nim, String prodi, String fakultas)
    {
        this.nama = nama;
        this.nim = nim;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    /* Getter dan Setter*/
    // Get name
    public String getNama()
    {
        return this.nama;
    }

    // Set name
   public  void setNama(String nama)
    {
        this.nama = nama;
    }

    // get nim
    public int getNim()
    {
        return this.nim;
    }

    // set nim
    public void setNim(int nim)
    {
        this.nim = nim;
    }

    // Get Prodi
    public String getProdi()
    {
        return this.prodi;
    }

    // Set Prodi
    public void setProdi(String prodi)
    {
        this.prodi = prodi;
    }

    // Get Fakultas
    public String getFakultas()
    {
        return this.fakultas;
    }

    // Set Fakultas
    public void setFakultas(String fakultas)
    {
        this.fakultas = fakultas;
    }
}
